### PR TITLE
Copy updates and section landmark labels

### PR DESCRIPTION
**Pull request by marcysutton**
Made a few copy updates. I'll probably have more as I go through Sanity, but those can be in another PR.

### CODE REVIEW

**Review by tayiorbeii — APPROVED** [review]
Looks good to me!

File: apps/testingaccessibility/src/components/content/new-landing-copy.mdx [hunk 1]
```diff
@@ -214,7 +214,13 @@ Other questions require a bit more thought and understanding of the nuances:
 
 </ListOfQuestions>
 
-It is critical that you can provide detailed explanations to ensure the collaborative design process is as inclusive as possible.
+<ListOfQuestions>
```
Comment (marcysutton) [inline apps/testingaccessibility/src/components/content/new-landing-copy.mdx:217]: I added another question here about user testing!

File: apps/testingaccessibility/src/components/content/new-landing-copy.mdx [hunk 2]
```diff
@@ -188,15 +188,15 @@ which helps me when I start thinking about how I'm going to solve accessibility
 
 </Section>
 
-<Section className="bg-moss-50 prose-green" image={require('../../../public/assets/lighthouse@2x.png')}>
+<Section aria-label="Accessibility in the design phase" className="bg-moss-50 prose-green" image={require('../../../public/assets/lighthouse@2x.png')}>
```
Comment (marcysutton) [inline apps/testingaccessibility/src/components/content/new-landing-copy.mdx:191]: Looks like these aria-labels aren't making it through as props.

Comment (vojtaholik) [reply apps/testingaccessibility/src/components/content/new-landing-copy.mdx:191]: will fix!

Comment (marcysutton) [reply apps/testingaccessibility/src/components/content/new-landing-copy.mdx]: I fixed it in the PR!

File: apps/testingaccessibility/src/components/pricing.tsx [hunk 3]
```diff
@@ -162,40 +162,43 @@ export const Pricing: React.FC<PricingProps> = ({
...
-              type="submit"
-              disabled={status === 'loading' || status === 'error'}
-            >
-              <span className="relative z-10">
-                {formattedPrice?.upgradeFromPurchaseId
-                  ? `Upgrade Now`
-                  : action || `Ship Accessible Apps Like a Pro`}
-              </span>
-            </button>
+            <fieldset>
```
Comment (marcysutton) [inline apps/testingaccessibility/src/components/pricing.tsx:165]: When I viewed the page locally the pricing section came up, so I made some improvements to that as well!

**Review by joelhooks — APPROVED** [review]

**Pull request merged**